src/ext/default.c

```diff
@@ -479,10 +479,49 @@ int StemmerExpander(RSQueryExpanderCtx *ctx, RSToken *token) {
     char *dup = rm_malloc(sl + 2);
     dup[0] = STEM_PREFIX;
     memcpy(dup + 1, stemmed, sl + 1);
+
+    // Get fieldMask which includes only expandable fields
+    QueryNode *qn = *ctx->currentNode;
+    t_fieldMask orig_fm = qn->opts.fieldMask;
+    t_fieldMask expandable_fm = qn->opts.fieldMask;
+    if (orig_fm != RS_FIELDMASK_ALL) {
+      t_fieldMask fm = qn->opts.fieldMask;
+      int i = 0;
+      while (fm) {
+        t_fieldMask bit = (fm & 1) << i;
```

Comment:
Left shift seems unnecessary (doing a right-shift below is enough?)

Reply:
The shift was needed because `bit` was used later as mask: `expandable_fm &= ~bit;`
But now I changed for a simpler bit masking without using `i`